example/slackchatapp/settings.py

```diff
@@ -79,9 +79,13 @@
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
 DATABASES = {}
-DATABASES['default'] = dj_database_url.parse(
-    'postgresql://postgres@localhost:5432/slackchat'
-)
+if 'DATABASE_URL' in os.environ:
+    DATABASES['default'] = dj_database_url.config()
+else:
+    DATABASES['default'] = {
```

Comment:
sqlite actually isn't a viable option anymore, right? Best to make this the default postgres we had.